Sources/TSCBasic/CMakeLists.txt

```diff
@@ -48,7 +48,9 @@ add_library(TSCBasic
   TerminalController.swift
   Thread.swift
   Tuple.swift
-  misc.swift)
+  misc.swift
+  WeakDictionary.swift)
```

Comment:
still in use, or leftover?

Reply:
`WeakDictionary.swift` was removed, so this will probably cause issues for bootstrapping using CMake.